src/sfizz/Region.cpp

```diff
@@ -1692,6 +1692,10 @@ bool sfz::Region::processGenericCc(const Opcode& opcode, OpcodeSpec<float> spec,
         auto it = std::find_if(connections.begin(), connections.end(),
             [ccNumber, &target](const Connection& x) -> bool
             {
+                if ((ccNumber > 130 && ccNumber < 138) || ccNumber == 140 || ccNumber == 141)
+                    return x.source.id() == ModId::PerVoiceController &&
+                        x.source.parameters().cc == ccNumber &&
```

Comment:
Shouldn't we check the region id here too in this case? If not, I'd suggest just modifying the initial check to `x.source.id() == ModId::PerVoiceController || x.source.id() == ModId::Controller`.

Reply:
Thanks for the critiques, Paul. Insofar as this code and the one with the corresponding suggestion, good thought about the region checking, I suppose. Beyond that technicality with this particular one, I'll let you be the judge over how to format the syntax exactly: I fully admit code optimization is not my forte.

Sorry for my delayed response, btw.